utils/bin/lvim

```diff
@@ -3,4 +3,36 @@
 export LUNARVIM_RUNTIME_DIR="${LUNARVIM_RUNTIME_DIR:-$HOME/.local/share/lunarvim}"
 export LUNARVIM_CONFIG_DIR="${LUNARVIM_CONFIG_DIR:-$HOME/.config/lvim}"
 
+headless_update() {
+  nvim --cmd 'lua _G.__lvim_updating=true' -u "$LUNARVIM_RUNTIME_DIR/lvim/init.lua" --noplugin -n -i NONE --headless
+}
+
+for arg; do
+  shift
+  case $arg in
+    --update)
+      :
+      echo 'Updating LunarVim...'
+      old="$(pwd)"
+      cd "$LUNARVIM_RUNTIME_DIR/lvim"
+      target_branch="${1:-$(git rev-parse --abbrev-ref HEAD)}"
+      git stash --include-untracked -m "Changes before LunarVim update" && echo "Uncommitted changes saved to stash" || true
+      git fetch --force --depth=1 origin $target_branch:$target_branch
```

Comment:
If you're starting with a shallow clone, then you won't be able to see any remote branch other the current. You can see the workaround I added in https://github.com/LunarVim/LunarVim/pull/2127/files#diff-592059d4c0b81bb5e3509fc175b0d838215ff52d6cfd5015ab13181b29f569deR37-R51

Reply:
That's not true

Reply:
![image](https://user-images.githubusercontent.com/5411/147876763-54b63709-fb19-41b1-9604-f7a9887d9aba.png)